docs/common/wibox.ldoc

```diff
@@ -180,6 +180,15 @@
 -- @property shape
 -- @tparam gears.shape shape
 
+--- Forward the inputs to the client below the wibox.
```

Comment:
I would mention that this is implemented by setting `shape_input` so that it becomes clearer that the two properties interact.